src/types.js

```diff
@@ -1,11 +1,17 @@
 /* @flow */
 
-export type Worker = {
+export type Target = {
```

Comment:
Maybe it should be like `Worker | ChromeRuntimePort`? Right now the type requires the methods to exist even if we don't use them